pkg/operator/k8sutil/statefulset.go

```diff
@@ -28,8 +28,13 @@ import (
 // make a headless svc for statefulset
 func CreateService(name, namespace string, clientset kubernetes.Interface, svc *corev1.Service) error {
```

Comment:
Could you call `CreateOrUpdateService()` (in service.go) and remove this `CreateService()` method? They look duplicated.

Reply:
@travisn done